ctags.go

```diff
@@ -222,37 +221,27 @@ func (p *ctagsProcess) Parse(name string, content []byte) ([]*Entry, error) {
 	}
 
 	if !utf8.Valid(content) {
-		if p.Info != nil {
-			p.Info.Printf("ctags skipping file due not being utf-8 encoded: %s", name)
-		}
-		return nil, nil
+		return nil, newParseError("file is not utf-8 encoded")
 	}
 
 	if ok, err := p.post(&req, content); err != nil {
 		return nil, err
 	} else if !ok {
-		if p.Info != nil {
-			p.Info.Printf("ctags skipping file due to long filename: %s", name)
-		}
-		return nil, nil
+		return nil, newParseError("filename is too long")
 	}
 
 	// 250 is a better guess for initial size
 	es := make([]*Entry, 0, 250)
 	for {
 		var rep reply
 		if err := p.read(&rep); err != nil {
-			return nil, err
+			return nil, &ParseError{Message: "error reading ctags reply", Fatal: true, Inner: err}
```

Comment:
I could just pass the error through here, but it seemed nice to always return a `ParseError`, and let the caller unwrap if they want.

Reply:
Before I saw this comment I actually commented that maybe its nicer to just wrap this error like normal rather than using ParseError since its always fatal in the case of error wrapping. But yeah, no strong opinion either way :)